pkgs/development/libraries/muparserx/default.nix

```diff
@@ -0,0 +1,31 @@
+{ stdenv
+, fetchFromGitHub
+, cmake
+  # Check Inputs
+, python
```

Comment:
unused
```suggestion
```

Reply:
TY. This made me remember I wanted to actually build & run its self-test module. I'll try to do that.